tests/BranchTypeConfigurationWrapperTest.php

```diff
@@ -29,38 +31,55 @@ public function setUp(): void
     }
 
     /**
-     * @return BranchTypeConfigurationAKVWrapper[][]|BranchTypeConfigurationKMSWrapper[][]
+     * @return BranchTypeConfigurationAKVWrapper[][]|BranchTypeConfigurationKMSWrapper[][]|BranchTypeConfigurationGKMSWrapper[][]
      */
     public function wrapperProvider(): array
     {
+        putenv('GOOGLE_APPLICATION_CREDENTIALS=' . getenv('TEST_GOOGLE_APPLICATION_CREDENTIALS'));
```

Comment:
Proc to tu nastavujes az v testu, kdyz AWS/AZ envy se konfiguruje uz v `setUp`?

Reply:
Protože je to potřeba už v GkmsClientFactory